src/bundle/Resources/public/scss/fieldType/preview/_ezimage.scss

```diff
@@ -5,4 +5,42 @@
         width: 50%;
         margin-right: 2rem;
     }
+
+    table {
```

Comment:
Please, don't tags as selectors. I prefer adding a CSS class that follows BEM convention to avoid complex CSS selectors.